addons/account_check_printing/migrations/9.0.1.0/post-migration.py

```diff
@@ -20,6 +20,28 @@ def update_payments_from_vouchers(env):
         WHERE ap.id = av.id
         """
     )
+    check_method = env.ref(
+        'account_check_printing.account_payment_method_check')
+
+    env.cr.execute("""SELECT column_name
```

Comment:
Next time, you can use https://doc.therp.nl/openupgrade/API.html#openupgradelib.openupgrade.column_exists